packages/genesys-spark-components/src/components/stable/gux-tabs/example.html

```diff
@@ -264,9 +264,104 @@ <h2>
     <gux-tab gux-disabled tab-id="10-4">Tab Header 4</gux-tab>
     <gux-tab gux-disabled tab-id="10-5">Tab Header 5 long long long</gux-tab>
   </gux-tab-list>
-  <gux-tab-panel-beta class="bg-color-example" tab-id="10-1"
-    ><div>Tab content 1</div></gux-tab-panel-beta
+  <gux-tab-panel-beta
+    class="bg-color-example"
```

Comment:
I noticed the elements with the `bg-color-example` class were removed in the linked PR for v4. Do we still want this class in v3?

Reply:
Yeah this is fine to keep in v3, it helps highlight what the flexbox attribute does